.travis.yml

```diff
@@ -21,13 +21,12 @@ env:
     - PIP_FIND_LINKS=file://$HOME/.cache/pip/wheels
 
 python:
-  - "2.7"
   - "3.6"
 
 addons:
   apt:
     packages:
-    - libgdal1h
+    - libgdal1i
     - gdal-bin
     - libgdal-dev
     - libatlas-dev
```

Comment:
@bhavika unless untiler makes use of the gdal-bin programs, I think our CI can use a rasterio wheel with GDAL included (same for numpy and libatlas-dev, &c) and all of these apt packages can go.

Reply:
@sgillies oh, thanks for explaining that! I can remove these then, I dont think untiler uses gdal-bin.

Reply:
You're welcome. This project's config could become as simple as the one in https://github.com/mapbox/rio-color/blob/master/.travis.yml.